lib/metadata/bucketfile/backend.js

```diff
@@ -0,0 +1,451 @@
+import net from 'net';
+import fs from 'fs';
+import cluster from 'cluster';
+import events from 'events';
+import assert from 'assert';
+
+import level from 'level';
+import multilevel from 'multilevel';
+import sublevel from 'level-sublevel';
+import arsenal from 'arsenal';
+
+import { logger } from '../../utilities/logger';
+import BucketInfo from '../BucketInfo';
+import constants from '../../../constants';
+
+const errors = arsenal.errors;
+
+const METADATA_PORT = 9990;
+const METADATA_PATH = '/metadata/';
```

Comment:
When you rebase and use the metadata path from the config, note that you are going to have to add a "/" to the end here.